tools/depends/target/libnfs/Makefile

```diff
@@ -3,12 +3,14 @@ DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=libnfs
-VERSION=2.0.0
+VERSION=4.0.0
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.gz
 
+BASE_URL := https://github.com/sahlberg/libnfs/archive
```

Comment:
This is wrong.

Reply:
Yeah, will update that, need to get the source tar on mirrors and drop this

Reply:
Hace it uploaded to our ftp and available now on http://mirrors.kodi.tv/build-deps/sources/libnfs-4.0.0.tar.gz

Reply:
Thanks Alwin, I'll fix this up today